examples/config/config

```diff
@@ -516,7 +517,7 @@ set ssl_policy fail
 
 # === Post-load misc commands ================================================
 spawn_sync_exec @scripts_dir/load_cookies.sh
-spawn_sync_exec @scripts_dir/load_cookies.sh @data_home/session-cookies.txt
+spawn_sync_exec @scripts_dir/load_cookies.sh @(echo "${UZBL_SESSION_COOKIE_FILE}")@
```

Comment:
I understand. Thank you for taking the time to explain :-).

I will try to think of and submit a solution to this issue.